tenets/codelingo/psr2/elseif-not-else-if/codelingo.yaml

```diff
@@ -3,6 +3,9 @@ funcs:
     type: resolver
     body: |
       function(rawCode) {
+        if (rawCode.substring(0, 7) !== "else if") {
+          return rawCode
```

Comment:
@Daanikus @Daanikus if I change `else if` https://github.com/codelingo/codelingo/blob/6177ac37edbec4058faaa5ac09d550e83c578f3a/tenets/codelingo/psr2/elseif-not-else-if/example.php#L8 

to `else` &nbsp;&nbsp;&nbsp;   `if` (more than one space), the tenet doesn't work